pkgs/top-level/all-packages.nix

```diff
@@ -6680,6 +6680,8 @@ with pkgs;
 
   ifcopenshell = with python3Packages; toPythonApplication ifcopenshell;
 
+  ifcfg = with python3Packages; toPythonApplication ifcfg;
+
```

Comment:
```suggestion
```
`ifcfg` appears to provide a `ifcfg.cli` module that can be used from the command line, but doesn't provide any binaries (in `bin`), and I don't think it needs to be part of the top-level package set.